packages/braid-design-system/src/lib/components/Accordion/Accordion.docs.tsx

```diff
@@ -220,6 +220,36 @@ const docs: ComponentDocs = {
           </>,
         ),
     },
+    {
+      label: 'Data attributes',
+      description: (
+        <>
+          <Text>
+            Braid components are very explicit about the properties they accept,
+            which makes providing arbitrary{' '}
+            <TextLink href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes">
+              data attributes
+            </TextLink>{' '}
+            not possible. Instead, all Braid components accept a{' '}
```

Comment:
It feels weird to read about 'all' components here. For the docs on individual components would it be better to just talk about this prop?

```suggestion
            not possible. Instead this component accept a{' '}
```

This is helpful for future flexibility (what if in the future one component doesn't allow data for some reason), and also (I think) reads better as it's not jumping to universally scoped information

Reply:
Yeah nice, how about:
```
Instead, a `data` prop can be provided as a single collection of data attributes.
```

Reply:
I like it. The less text the better as people will be seeing it on every page so the easier it is to see and say "oh yeah, I've read the before" the better.